packages/styles/scss/components/logo-grid/_logo-grid.scss

```
@@ -83,7 +84,9 @@
     }
   }
 
-  .#{$prefix}--logo-grid__container {
+  .#{$prefix}--logo-grid__container,
+  :host(#{$dds-prefix}-logo-grid):not(.#{$prefix}--logo-grid__no-border)
```

We don't use CSS classes on our own Web Components, given it tends to clash with CSS classes application code sets:

```suggestion
  :host(#{$dds-prefix}-logo-grid):not([hide-border])
```